optimum/neuron/accelerate/accelerator.py

```diff
@@ -191,61 +190,47 @@ def _prepare_optimizer_for_zero_1(self, optimizer: torch.optim.Optimizer, device
             defaults = args_and_kwargs_to_kwargs_only(optimizer.__class__, args[1:], kwargs)
-
-            # Prior to 1.13.1+torchneuron8, the vanilla ZeroRedundancyOptimizer was not designed for TP.
-            full_torch_xla_version = get_torch_xla_version()
-            torch_xla_version, torchneuron_version = full_torch_xla_version.split("+")
-            torch_xla_version = version.parse(torch_xla_version)
-            if torch_xla_version <= version.parse("1.13.1") and int(torchneuron_version[-1]) < 8:
-                zero_1_optimizer = ZeroRedundancyOptimizerCompatibleWithTensorParallelism(
-                    params,
-                    optimizer.__class__,
-                    optimizer_dtype=optimizer_dtype,
-                    pin_layout=False,
-                    **defaults,
-                )
-            # Exception to make sure `ZeroRedundancyOptimizerCompatibleWithTensorParallelism` is removed in 3 releases.
-            elif torch_xla_version <= version.parse("1.13.1") and int(torchneuron_version[-1]) >= 11:
-                raise RuntimeError(
-                    "ZeroRedundancyOptimizerCompatibleWithTensorParallelism is deprecated and should be removed from "
-                    "`optimum-neuron`"
-                )
-            else:
-                from neuronx_distributed.parallel_layers.parallel_state import (
-                    get_data_parallel_group,
-                    model_parallel_is_initialized,
-                )
-                from torch_xla.distributed.zero_redundancy_optimizer import ZeroRedundancyOptimizer
-
-                if not is_neuronx_distributed_available() or not model_parallel_is_initialized():
-                    cc_op_groups = None
-                else:
-                    cc_op_groups = get_data_parallel_group(as_list=True)
-
-                zero_1_optimizer = ZeroRedundancyOptimizer(
-                    params,
-                    optimizer.__class__,
-                    optimizer_dtype=optimizer_dtype,
-                    pin_layout=False,
-                    cc_op_groups=cc_op_groups,
-                    **defaults,
-                )
+            zero_1_optimizer = NeuronZero1Optimizer(
+                params,
+                optimizer.__class__,
+                optimizer_dtype=optimizer_dtype,
+                pin_layout=False,
+                sharding_groups=sharding_groups,
+                grad_norm_groups=grad_norm_groups,
+                **defaults,
+            )
             del optimizer
         else:
             logger.warning(
                 f"Creating a ZeroRedundancyOptimizer from {optimizer}, this might change some default values. When "
                 "using ZeRO 1 it is recommended to create the ZeroRedundancyOptimizer yourself to avoid this kind of "
                 "issues."
             )
-            zero_1_optimizer = ZeroRedundancyOptimizerCompatibleWithTensorParallelism(
+            zero_1_optimizer = NeuronZero1Optimizer(
```

Comment:
You need t modify the text of the warning above also `ZeroRedundancyOptimizer` -> `NeuronZero1Optimizer`